infra/test/integration/simple_example/simple_example_test.go

```diff
@@ -58,6 +58,39 @@ func TestSimpleExample(t *testing.T) {
 			return false, nil
 		}
 		utils.Poll(t, isServing, 10, time.Minute*1)
+
+		// Check cloud run jobs status
+		isJobCompletion := func() (bool, error) {
+			cloudRunJobs := gcloud.Run(t, ("beta run jobs list --format=json"), gcloudArgs).Array()
+			var jobResults []bool
```

Comment:
question: wouldn't we want to store job results by name? If we have 2 out of 5 jobs failing, we'll want to know which jobs failed and why.

Reply:
In this case, the test serves as an indicator which helps to ensure that the job is not in an active state before proceeding with CFT teardown.